src/App.jsx

```diff
@@ -1,28 +1,50 @@
 import React from 'react';
+
 import './App.scss';
 import './styles/general.scss';
+
+import { getTodos } from './api/todos';
+
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
 class App extends React.Component {
   state = {
     todos: [],
-    selectedUserId: 0,
+    onSelectedUserId: 0,
   };
 
+  componentDidMount() {
+    getTodos()
+      .then((todos) => {
+        this.setState({ todos });
+      });
+  }
+
   render() {
-    const { todos, selectedUserId } = this.state;
+    const { todos, onSelectedUserId } = this.state;
 
     return (
       <div className="App">
         <div className="App__sidebar">
-          <TodoList todos={todos} />
+          <TodoList
+            todos={todos}
+            onSelectedUserId={onSelectedUserId}
+            onUserSelected={(userId) => {
+              this.setState({ onSelectedUserId: userId });
+            }}
```

Comment:
it's better to create a separate method and pass it as a callback, in that way you can use some readable name, which will help everyone to read your code.